packages/angular-ui-components/README.md

```diff
@@ -109,9 +109,27 @@ In addition, you can customize the theme by changing some CSS properties.
 }
 ```
 
+## Caching
+
+By default, the `KvIcon` and `KvIllustration` components require an SVG file with all the `kv-icons` available. The default file is `svg-symbols.svg` which is provided after installing this dependency. For caching purposes, it is also provided a `symbols.${checksum}.svg` file after installation. If you are caching those SVGs in your project you should provide the latter to the library configuration in your `app.component.ts`.
+
+```tsx
```

Comment:
minor detail but use typescript md as the example is for angular which doesn't use JSX.

Reply:
Done :muscle: